app/services/request_referer_service.rb

```diff
@@ -19,14 +19,11 @@ def self.access_whitelisted?(request, controller_name, action_name)
     'vm'                       => %w(launch_html5_console show:id show_list), # http://localhost:3000/vm/show/400r6
     # 'launch_html5_console' above added due to IE even in version 11 does not set the referrer headir
     # most likely when the new window is opened by JavaScript code
-    'availability_zone'        => %w(show:id show_list), # http://localhost:3000/availability_zone/show/400r1
     'host'                     => %w(show:id show_list), # http://localhost:3000/host/show/400r6
     'miq_request'              => %w(show:id show_list), # http://localhost:3000/miq_request/show/400r3
     'ems_cluster'              => %w(show:id show_list), # http://localhost:3000/ems_cluster/show/400r2
     'storage'                  => %w(show:id show_list), # http://localhost:3000//storage/show/400r1
     'ems_container'            => %w(show:id show_list), # http://localhost:3000/ems_container/show/400r1
-    'flavor'                   => %w(show:id show_list),
-    'cloud_tenant'             => %w(show:id show_list), # http://localhost:3000/cloud_tenant/show/400r1
```

Comment:
@chessbyte : we are reverting a previous change for the very reason that you mentioned

I overlooked it previously :-(